op-e2e/e2eutils/disputegame/super_game_helper.go

```diff
@@ -43,6 +42,10 @@ func NewSuperGameHelper(t *testing.T, require *require.Assertions, client *ethcl
 				}
 				return fmt.Sprintf("Timestamp: %v, Step: %v", timestamp, step)
 			},
+			ClaimedL2SequenceNumber: func(pos types.Position) (uint64, error) {
+				timestamp, _, err := correctOutputProvider.ComputeStep(pos)
+				return timestamp, err
+			},
```

Comment:
I think this is wrong - in super cannon games the claimed L2 sequence number is always the one from the game proposal, regardless of the position in the game.

From https://specs.optimism.io/interop/fault-proof.html#overview
> The L2 block number preimage oracle local key now always provides the timestamp of the proposal.

Reply:
Sadly I don't think fixing this fixed the test.

Reply:
The `ClaimedL2SequenceNumber` is a misleading name. it should be named `timestampAtPosition` and it's only used to navigate the top game. Though correct me if I'm off base here.
The tests seem to be failing for some other reason.

Reply:
Ah yeah I think you're right. Ignore me.